### PR TITLE
Events Page

### LINKED ISSUE #2822: create overview page for webinars

### PR DESCRIPTION
**Pull request by Cahllagerfeld**
closes #2822 

https://deploy-preview-3010--gitpod-kumquat.netlify.app/events

- [x] optimize mobile
- [x] add remaining links
- [x] Dark Theme images

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3010"></a>

### CODE REVIEW

**Comment by Cahllagerfeld** [comment]
Hey @chrifro, from my side it's ready. One small nit, the rendering of the skateboard, and especially the downside of the deck looks a bit noisy (here and in figma as well) Do you think that's working out, or do we need to re-render it with higher specs? I assume it was made with Blender and the Cycles engine, so increasing the render samples might do the trick 🤔

**Review by chrifro — APPROVED** [review]
Awesome, thanks @Cahllagerfeld 

>the rendering of the skateboard, and especially the downside of the deck looks a bit noisy

Totally, I'll look into it 

>Where should we put a reference to the new page? 


Let's add it to the footer under "resources"

**A few more comments/thoughts:**
- How can we make the process of remembering to add new webinars and events to this page as easy as possible?

- [x] The section "highlights from DevX Conf" seems a bit random after the webinars. Let's move it  including the devx call-out underneath the conferences

- [x] mobile: reduce spacing -> does the new [mobile spacing overview](https://www.figma.com/file/VL0CJpHqcDJVJ4BEkoyRFV/Gitpod-Website-Component-Library?node-id=2864%3A2856&t=V2VZDWBwW6Iz7gms-1) make sense here?
Bildschirmfoto 2022-11-10 um 16 15 22

File: src/lib/components/events/landing-page/more-about-devx.svelte [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  >
+    <div class="flex flex-col justify-between w-full">
+      <h3>More about DevX</h3>
+      <p class="text-left mb-x-small">
+        To learn more about our involvement in the world of developer
+        experience, check out the DevX Community and previous conferences.
+      </p>
+      <div class="flex gap-micro">
+        <LinkButton href="#" variant="primary" size="medium"
+          >DevX Community</LinkButton
```
[the remark this answers is not in the document]
Comment (Cahllagerfeld) [reply src/lib/components/events/landing-page/more-about-devx.svelte:19]: Nice catch! I forgot to replace the dummy

**Comment by Cahllagerfeld** [comment]
For making it as easy as possible to remember, is maybe adding a note in the `webinars`-directory, so whenever a new page gets created there people will see that note? 🤔

Other than that I adjusted the spacings to the new mobile ones.

Do you think we should merge and adjust the skateboard image in a later iteration, or should we wait with merging until the skateboard is in palce? :)

File: src/lib/contents/events.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  {
+    text: "Join Kirill Skrygan, Christian Weichel, Nik Molnar, Christof Marti and Johannes Landgraf for a discussion on remote development.",
+    title: "Remote development panel",
+    link: {
+      href: "https://www.youtube.com/watch?v=tWZ6VunABws",
+      text: "Watch on Youtube",
+    },
+  },
+  {
+    text: "DevX has been an emerging concept for the past two years, and while we are still trying to give it a shared and structured definition in the tech industry, some common questions are already arising, and the most compelling one is: where to start with developer experience?",
```
[the remark this answers is not in the document]
Comment (chrifro) [reply src/lib/contents/events.ts:46]: Did you implement it? I can't see it in the preview
Bildschirmfoto 2022-11-11 um 11 23 20

[the remark this answers is not in the document]
Comment (chrifro) [reply src/lib/contents/events.ts:46]: Looks like my comment move to the wrong code snippet, I was talking about this section
Bildschirmfoto 2022-11-11 um 11 23 20

**Comment by chrifro** [comment]
>For making it as easy as possible to remember, is maybe adding a note in the webinars-directory, so whenever a new page gets created there people will see that note? 🤔

Sounds good. Let's do that 👍

The skateboard image has been updated, see[ Figma ](https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj?node-id=7230:12039#305334481).

Feel free to merge once that's updated. Thank you! 🤸‍♀️

**Pull request merged**